docs/development/building.md

```diff
@@ -1,182 +1,131 @@
 # Building Akri Containers
 
-Building Akri containers, whether locally or in the automated CI builds, leverages the same set of `make` commands.
+Building Akri containers, whether locally or in the automated CI builds, leverages the same set of Dockerfiles. In order to help with local development, a set of `Makefile` exists.
 
-In essence, Akri components can be thought of as: 
+The Makefiles are using `docker buildx` behind the scenes, ensure you have it installed, if you want to build for foreign architectures, you must also ensure you have correctly set up your docker builder to do so (see [Docker buildx documentation](https://github.com/docker/buildx#building-multi-platform-images))
 
-1. Runtime components 
+In essence, Akri components can be thought of as:
+
+1. Runtime components
     1. Rust code: containers based on Rust code are built using `Cargo cross` and subsequent `docker build` commands include the cross-built binaries.
-        > Note: For Rust code, `build/Dockerfile.*` does NOT run `cargo build`, instead they simply copy cross-built binaries into the container 
-    2. Other code: these containers can be .NET or python or whatever else ... the `build/Dockerfile.*` must do whatever building is required. 
+        > Note: For Rust code, `build/Dockerfile.*` does NOT run `cargo build`, instead they simply copy cross-built binaries into the container
+    2. Other code: these containers can be .NET or python or whatever else ... the `build/Dockerfile.*` must do whatever building is required.
 2. Intermediate components: these containers are used as part of the build process and are not used in production explicitly
 
-## Runtime components
+## Akri components
+
+The Akri core components are the containers that provide Akri's functionality. They include the agent, the controller, the webhook and the discovery handlers. All of these are written in Rust.
 
-The Akri runtime components are the containers that provide Akri's functionality. They include the agent, the controller, the webhook, the brokers, and the applications. The majority of Akri runtime components are written in Rust, but there are several components that are written in .NET or python.
+The samples containers are a set of brokers and applications written either in .NET, python or Rust that are used in this documentation examples.
```

Comment:
As written currently, it sounds like only those three langs are permitted broker languages. How about?
```suggestion
The samples containers are a set of brokers and applications that can be written in any language, such as .NET, python or Rust. They are used in documentation examples, quickstarts, and demos.
```